scripts/utils/generate_package_wiki.py

```diff
@@ -0,0 +1,62 @@
+import os
+import sys
+import re
+import datetime
+import argparse
+
+
+packagesByCategory={}
+
+
+def write_wiki_content(file_path):
+    wikiContent = ""
+    for category in packagesByCategory.keys():
+        wikiContent += "## " + category + "\n\n"
+        wikiContent += "| Package | Description |\n"
+        wikiContent += "|---|---|\n"
+        wikiContent += packagesByCategory[category] + "\n\n"
```

Comment:
I find it confusing that the table format is defined in two different functions. This is an issue when adding new items to the table. :thinking:

Reply:
yes,  that is because each value in the dictionary contains the package name and the description already formatted in markdown previously. It is not possible to add the markdown headers then as this can only be done when all the packages have been processed. It is done in the `write_wiki_content` also because dictionaries are sorted alphabetically by default.

Reply:
yes, but the current implementation makes it difficult to understand it. This can be enhanced for example in one of the following ways:
- `packagesByCategory` contains a hash or tuple so that you keep the logic from parsing the packages and the logic from creating markdown in two separate functions
- With the other code suggestions, the resulting code may be simpler enough that we can have all the logic in the same function.

In any case, documentation in the code (as function descriptions and comments) is needed.

Reply:
I feel with the added documentation, it is ok to keep the current code structure.